libraries/Microsoft.Bot.Builder.AI.QnA/Helpers/HttpRequestHelper.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Reflection;
+using System.Runtime.Versioning;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Bot.Builder.AI.QnA
+{
+    /// <summary>
+    /// Helper for HTTP requests.
+    /// </summary>
+    public class HttpRequestHelper
+    {
+        private readonly HttpClient _httpClient;
```

Comment:
This should be an HTTP Client Handler:
https://docs.microsoft.com/en-us/previous-versions/visualstudio/hh138157(v=vs.118)

Reply:
Using HTTP Client handler will be a breaking change as in QnA Maker constructor, we give user option to provide their HTTP CLient object. We can make this change in the next version as this version is mostly about injecting active learning stuff and a little bit of code refactoring to incorporate that.